core/src/main/java/org/jdbi/v3/core/mapper/ColumnMappers.java

```diff
@@ -87,6 +88,32 @@ public ColumnMappers register(ColumnMapperFactory factory) {
      * @param config the configuration registry
      * @return a ColumnMapper for the given type, or empty if no column mapper is registered for the given type.
      */
+    @SuppressWarnings("unchecked")
+    public <T> Optional<ColumnMapper<T>> findFor(Class<T> type, ConfigRegistry config) {
+        ColumnMapper<T> mapper = (ColumnMapper<T>) findFor((Type) type, config).orElse(null);
+        return Optional.ofNullable(mapper);
+    }
+
+    /**
+     * Obtain a column mapper for the given type.
+     *
+     * @param type the target type to map to
+     * @param config the configuration registry
+     * @return a ColumnMapper for the given type, or empty if no column mapper is registered for the given type.
+     */
+    @SuppressWarnings("unchecked")
+    public <T> Optional<ColumnMapper<T>> findFor(GenericType<T> type, ConfigRegistry config) {
+        ColumnMapper<T> mapper = (ColumnMapper<T>) findFor(type.getType(), config).orElse(null);
```

Comment:
It's a generics thing. The compiler won't let me cast from `Optional<ColumnMapper<?>>` to `Optional<ColumnMapper<T>>`. But if I unwrap the `ColumnMapper<?>`, it _will_ let me cast that to `ColumnMapper<T>`. I think javac just gives up on nested wildcards.